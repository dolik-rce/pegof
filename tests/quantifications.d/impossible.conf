input quantifications.d/impossible.peg
optimize repeats
