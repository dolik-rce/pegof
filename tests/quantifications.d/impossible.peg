Z <-
    B1 C1
    / B2 C2

X <- "X"

B1 <- X+ X

C1 <- "A" / X* X

B2 <- X+ X+ / "B"

C2 <- "A" / X* X+ / "B"
