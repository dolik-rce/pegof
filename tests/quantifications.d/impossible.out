WARNING: Detected sequence that will never match: X+ X
Z <-
    B1 C1
    / B2 C2

X <- "X"

B1 <- X+ X

C1 <- "A"

B2 <- "B"

C2 <-
    "A"
    / "B"
